### PR TITLE
Rework exceptions in main

### PR DESCRIPTION
**Pull request by OznOg**
Rework exceptions in the main.cc file.

### CODE REVIEW

**Comment by OznOg** [comment]
any chance to get a review sometime?

**Comment by ooshlablu** [comment]
@OznOg After 1.2, I'll update the branch with master and build some packages and see if I notice any weird behavior.

**Comment by OznOg** [comment]
FYI, there is a lot of reindent, if you ca, look ate the diff ignoring whitespaces

**Comment by Baklap4** [comment]
> FYI, there is a lot of reindent

Happens when a try-catch block gets removed/replaced hahah

File: game/main.cc [hunk 1]
```diff
@@ -130,116 +99,106 @@ void mainLoop(std::string const& songlist) {
...
 	Songs songs(database, songlist);
 	loadFonts();
-	try {
-		window = std::make_unique<Window>();
-		} catch (RUNTIME_ERROR& e) {
-			std::cerr << "ERROR: " << e.what() << std::endl;
-		}
-	Game gm(*window, audio, localization);
+
+        Window window{};
```
Comment (ooshlablu) [inline game/main.cc:108]: I think this is indented too far

File: game/main.cc [hunk 2]
```diff
@@ -291,24 +248,16 @@ template <typename Container> void confOverride(Container const& c, std::string
...
+static void fatalError(const std::string &msg) {
+	auto errMsg = msg + "\nIf you think this is a bug in Performous, please report it at \n"
+	                    "  https://github.com/performous/performous/issues";
+        auto title = "FATAL ERROR";
+	SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, title, errMsg.c_str(), nullptr);
 	std::cerr << title << ": " << msg << std::endl;
-	if (hasLog) {
-		std::clog << "core/error: " << errMsg.str() << std::endl;
-	}
+        std::clog << "core/error: " << errMsg << std::endl;
```
Comment (ooshlablu) [inline game/main.cc:257]: I think this line is also indented too far

**Review by ooshlablu — CHANGES_REQUESTED** [review]
I think there are a couple of lines indented too far. I may be wrong

File: game/main.cc (hunk 2 above)
Comment (OznOg) [reply game/main.cc]: done

File: game/main.cc (hunk 1 above)
Comment (OznOg) [reply game/main.cc]: done

**Review by ooshlablu — APPROVED** [review]
I just tested singing on Ubuntu 18.04 & 20.04 and everything appears to work as expected. I did not have time to test guitar and guitar + singing though

**Review by Baklap4 — APPROVED** [review]
LGTM

**Pull request merged**